ethereum/src/ERC20Manager.sol

```diff
@@ -419,7 +419,7 @@ contract ERC20Manager is
             revert InvalidSender();
         }
 
-        if (_knownVftManagers.contains(source)) {
+        if (_vftManagers.contains(source)) {
```

Comment:
_governancePauser source check?

Reply:
so we may need to pause erc20manager (except for incoming governance accesses)

Reply:
it's handled in different way: governance pauser (Vara) -> GovernancePauser (ETH) -> MessageQueue -> ERC20Manager